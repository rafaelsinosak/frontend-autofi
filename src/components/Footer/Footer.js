import AppBar from "@material-ui/core/AppBar";
import { Container, Toolbar, Typography } from "@material-ui/core";
import moment from "moment";
import "./Footer.css";

const Footer = () => {
  return (
    <AppBar position="static" color="primary">
      <Container maxWidth="md">
        <Toolbar>
          <Typography variant="body1" color="inherit">
            Â© {moment().format("YYYY")} AutoFi
          </Typography>
        </Toolbar>
      </Container>
    </AppBar>
  );
};

export default Footer;
