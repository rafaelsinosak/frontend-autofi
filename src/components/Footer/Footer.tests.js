import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

test('Footer - Render Component', () => {
  const component = renderer.create(
    <Footer></Footer>,
  );
  let footer = component.toJSON();
  expect(footer).toMatchSnapshot();
});