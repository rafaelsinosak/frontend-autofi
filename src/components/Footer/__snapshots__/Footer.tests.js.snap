// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer - Render Component 1`] = `
<header
  className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
>
  <div
    className="MuiContainer-root MuiContainer-maxWidthMd"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorInherit"
      >
        Â© 
        2021
         AutoFi
      </p>
    </div>
  </div>
</header>
`;
